pdns/dnsname.cc

```diff
@@ -24,6 +24,7 @@ std::ostream & operator<<(std::ostream &os, const DNSName& d)
 DNSName::DNSName(const char* p)
 {
   d_empty=false;
+  d_recurse=0;
```

Comment:
we have a d_recurse in DNSName??? this is an abomination. We store hundreds of millions of DNSNames, none of which need d_recurse in storage.